pokemongo_bot/cell_workers/pokemon_catch_worker.py

```diff
@@ -185,6 +186,8 @@ def work(self):
                                 if status is 3:
                                     logger.log(
                                         'Oh no! {} vanished! :('.format(pokemon_name), 'red')
+                                    if success_percentage == 100:
+                                        self.softban = True
```

Comment:
@douglascamata , I am not sure this works. It's better be `self.bot.softban = True`